doc: Create blog submission template (#138)
WomenCodingCommunity/WomenCodingCommunity.github.io: .github/ISSUE_TEMPLATE/blog_submission.md

---
name: Blog submission request
about: Submit one blog post to be published in our website
title: 'blog: '
labels: 'blog'
assignees: ''

---

### Blog Metadata

Please provide the following information:

* Title:

> * Author:

Reviewer: I think it would be helpful to include position / company as well, what do you think?

> * Date:

Author: ```suggestion
* Position / Company: 
* Date:
```

* Short description:
* Google doc Link with the content:
* Category

### Banner image

> Please upload banner image for your post as a suggestion.

Author: ```suggestion
Please include the banner image link for your post as a suggestion.
```

Keep in mind the image should be free or your own image creation.

> In case you don't have any, the team can suggest.

Reviewer: For the banner, we might suggest to attach a link with the image (so the quality will remain the same).

> Is there other images you want to include the content? Please include here also.

Author: ```suggestion
Is there other images you want to include the content? Please include their links here also.
```
